### PR TITLE
Evfi disk areas: evaluate for merging

### PR DESCRIPTION
**Pull request by evezeyl**
Added a Memo based on what I read in README from Abel ... like a to do list

QQQ: are symbols for my questions (easier to find)

I tried to synthesize from what I read now - also on Abel user guide, using the beginner mindset to explain :)

Figure done in google docs with the google draw, easy to transfer/share

Let me know ... modifications, osv...

### CODE REVIEW

File: disk_areas.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+
+Abel Disk areas that are relevant for your work at the Veterinary Institute
+
+# Summary :
+
+|directory|path                |what      |characteristics|
+|:--------|:-------------------|:---------|:--------------|
+|`$HOME`|`usit/abel/u1/username`|"home-home" area|500GB - login area- NOT FOR Data Processing -backed-up|
+|**work**|`/work/projects/nn9305k`|common resources (eg. softwares, databases)|10 TB - faster disc, closer to nodes connection|
+|**project**|`/projects/nn9305k`|Project data area (backed-up, raw data, results)|30 TB - slower disc|
```
Comment (karinlag) [inline disk_areas.md]: Also project home, for the /projects/nn9305k/home/username area

File: disk_areas.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+|`$SCRATCH`|`/work/jobs/JOBID.d`|computing nodes: **FOR PROCESSING DATA = RUNNING JOBS**|fastest|
```
Comment (karinlag) [inline disk_areas.md]: Temp area on the compute nodes, store for lifetime of job only

File: disk_areas.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+|`$USERWORK`|`/work/users/username`|sharing data among different jobs = **staging**|do not backed-up - removed after 45 days|
+
+1. **$HOME**: "home-home" directory (on login nodes): 500GB - backed up regularly: slow and costly.
```
Comment (karinlag) [inline disk_areas.md]: costly how? it is free, money wise?

File: disk_areas.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  - If you need to store files that do not require backup: put them in a `nobackup` directory
+  - recommended for: software, job configurations, job preparation, debugging ...
+  - DO NOT USE to run jobs => long jobs should be run on $SCRATCH (submitted before hand to the queue system **SLURM**)
+
+QQQ: -> **login nodes** NB: small tests are tolerated (processes taking more than 30 min are automatically killed) automatically <=> so in home-home
```
Comment (karinlag) [inline disk_areas.md]: this goes for any location.

File: disk_areas.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+- softwares and libraries
+- public reference data, template files, adapters files ...
+
+2. **project area**: `/projects/nn9305k` will be used to contain:
+ - project data, including raw sequencing files (zipped/tared:compressed and decompressed)
+ - your `project/home/username`
+ - data analyses that have been performed
+ - ... this is where the work will NOW be done.
+
+QQQ - [ ]-> **so our(vetinst users) /work/projects/nn9305k/home/username (incl. all subfolder should be moved here)** -> `project/home/username`
```
Comment (karinlag) [inline disk_areas.md]: yes

File: disk_areas.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+3. **$SCRATCH**: when a job is started, a temporary directory on $SCRATCH is automatically created. It is also automatically deleted when the job finishes. This allows the jobs to run faster without interfering with the work of others. You can access $SCRATCH for status monitoring of running jobs: with `/work/jobs/JOBID.d` where JOBID.d is the ID that has been provided when you submitted the job in the queue system SLURM.
+
+4. **$USERWORK** If files are needed for more than one job, they are **staged** in here. Files are automatically deleted after 45 days, and there is no backup. Access: `/work/users/username`
```
Comment (karinlag) [inline disk_areas.md]: you should put your files there, and run analysis from that area, either via slurm script or other mechanisms.

File: disk_areas.md (hunk 3 above)
Comment (evezeyl) [reply disk_areas.md]: time wise

File: disk_areas.md (hunk 6 above)
Comment (evezeyl) [reply disk_areas.md]: other mechanisms ? qlogin? command line? ...maybe will also find out later on

File: disk_areas.md (hunk 1 above)
Comment (evezeyl) [reply disk_areas.md]: done

File: disk_areas.md (hunk 3 above)
Comment (evezeyl) [reply disk_areas.md]: done corrected

File: disk_areas.md (hunk 4 above)
Comment (evezeyl) [reply disk_areas.md]: ok - moved at the end: NB

File: disk_areas.md (hunk 5 above)
Comment (evezeyl) [reply disk_areas.md]: ok :)

File: disk_areas.md (hunk 6 above)
Comment (evezeyl) [reply disk_areas.md]: added to text

**Comment by evezeyl** [comment]
Corrected all the parts we discussed yesterday. Rechecked today. 
Think ready to merge.

File: disk_areas.md [hunk 7]
```diff
@@ -30,7 +30,9 @@ Abel Disk areas that are relevant for your work at the Veterinary Institute
 
 5. **$SCRATCH**: when a job is started, a **temporary area** on $SCRATCH is automatically created. It is also automatically deleted when the job finishes. This allows the jobs to run faster without interfering with the work of others. You can access $SCRATCH for status monitoring of running jobs: with `/work/jobs/JOBID.d` where JOBID.d is the ID that has been provided when you submitted the job in the queue system SLURM (thus accessible only during job's lifetime).
 
-6. **$USERWORK** If files are needed for more than one job, they are **staged** in here. Files are automatically deleted after 45 days, and there is no backup. Access: `/work/users/username`. You should put files required for analysis here and run analysis from this area (via SLRUM script or other mechanisms).  
+6. **$USERWORK** If files are needed for more than one job, they are **staged** in here. Files are automatically deleted after 45 days, and there is no backup. Access: `/work/users/username`. You should put files required for analysis here and run analysis from this area (via SLRUM script or other mechanisms).
```
Comment (karinlag) [inline disk_areas.md]: May I suggest that you begin with saying that while Bifrost puts files there itself, for other jobs have to put the files there themselves? That way we begin with how they get there, and then we say how they are deleted.

File: disk_areas.md [hunk 8]
```diff
@@ -39,10 +41,13 @@ For more details you can look at: [Managing Data on Abel] and more generally at
 <img src="https://docs.google.com/drawings/d/e/2PACX-1vSY_KCj3fubTH1zk6ZkOL6eLhoOOuAbp4bfu1YkOAvkadHhPfbuZrsepwHCUpEqwr45Zqt2hlEoCwVk/pub?w=960&amp;h=720">
```
Comment (karinlag) [inline disk_areas.md]: I LOVE this figure, however, it a little bit mixes disk areas with compute nodes/slurm. Not sure how to sort that out yet. Maybe discuss on wednesday?

File: disk_areas.md [hunk 9]
```diff
@@ -39,10 +41,13 @@ For more details you can look at: [Managing Data on Abel] and more generally at
...
 
 **SLURM** is the queue system and scheduler daemon (program/process which always runs in the background on Abel). It allows optimizing computing resources and schedule when jobs are to be run on $SCRATCH.- **All long jobs should be sumbited to to the queue system SLURM (and are run on $SCRATCH)**. 
+> There are 2 ways to use slurm SLURM queue system: 1. **interactively** with the command: `qlogin` and 2. by **submitting a script**. 
+> - when you use `qlogin` you actually are logged on a computing node that now executes commands from the shell. Usage: small tests
```
Comment (karinlag) [inline disk_areas.md]: Usage: small tests, to test out a working process, or if you have interactive programs.

**Comment by karinlag** [comment]
Looks good, had two small comments, but other than that, ready to merge.

File: disk_areas.md (hunk 8 above)
Comment (evezeyl) [reply disk_areas.md]: ok - splitted

File: disk_areas.md (hunk 9 above)
Comment (evezeyl) [reply disk_areas.md]: ok - added

File: disk_areas.md (hunk 7 above)
Comment (evezeyl) [reply disk_areas.md]: ok -done

**Comment by evezeyl** [comment]
Ok - added modifications according to comments + splitted figures in storage areas and computing areas. You can look, if further modification to figs need to be done, will not affect merged file - so I merge

**Pull request merged**